scripts/pipeline.test.ts

```diff
@@ -0,0 +1,72 @@
+import { fail } from "assert";
+
+import axios, { AxiosResponse } from "axios";
+
+
+const APP_NAME =
+  process.env.APP_NAME
+
+const TEST_URL =
+  process.env.TEST_URL
+
+describe("Smoke Test", () => {
+  describe(`Test case: ${APP_NAME} endpoints`, () => {
+    test(`Test http redirect (${APP_NAME})`, async () => {
+      try {
+        const url = `http://${TEST_URL}/`;
+        const response: AxiosResponse = await axios
+          .request({
+            method: "GET",
+            url: url,
+            headers: {
+              Accept: "application/json, text/plain, */*",
+              "Accept-Encoding": "gzip",
+            },
+            maxRedirects: 0,
+            validateStatus: false,
+          })
+
+        expect(response.status).toBe(301);
+        expect(response.headers.location).toBe(
+          `https://${TEST_URL}/`
+        );
+      } catch(error) {
+        fail(error.stack);
+      }
+    });
+    test(`Test http response (${APP_NAME}/health)`, async () => {
+      try {
+        const url = `https://${TEST_URL}/health`;
+        const response: AxiosResponse = await axios
+          .request({
+            method: "GET",
+            url: url,
+            headers: {
+              Accept: "application/json, text/plain, */*",
+              "Accept-Encoding": "gzip",
+            },
+          })
+
+        expect(response.data.status).toBe("UP");
+      } catch(error) {
+        fail(error.stack);
+      }
+    });
+    test(`Test http response (${APP_NAME}/)`, async () => {
+      try {
+        const url = `https://${TEST_URL}/`;
+        const response: AxiosResponse = await axios
+          .request({
+            method: "GET",
+            url: url,
+            headers: {
+              "Accept-Encoding": "gzip",
+            },
+          })
+        expect(response.data).toContain("There are no recipes");
```

Comment:
this shouldn't be asserted as someone could add a recipe, a 200 response should probably be enough